utils/format.js

```diff
@@ -20,3 +20,7 @@ export function formatDateTime(secondsSinceEpoch) {
   };
   return d.toLocaleDateString('en-US', options);
 }
+
+export function capitalizeFirstLetter(string) {
```

Comment:
can we rename to `capitalize` to match other utility libraries like lodash.

Reply:
usually "uppercase" is used when making all letters capital, capitalize means make the first letter uppercase.

Reply:
Fixed on commit [779b1b0](https://github.com/stripe-samples/treasury-nextjs/pull/19/commits/779b1b01fe53154e3ad6d07714f1c83c51607c17)